modules/installation-user-defined-tags-azure.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+// * installing/installing_azure/installing-azure-customizations.adoc
+
+:_content-type: CONCEPT
+[id="installing-azure-user-defined-tags_{context}"]
+= User-defined tags for Azure
+
+In {prodcut-ttitle} you can use the Azure user-defined tags for grouping resources and for managing resource access and cost. You can define tags on the Azure resources in the `install-config.yaml` file only during {product-title} cluster creation. You cannot modify the user-defined Azure tags after cluster creation.
+
+Support for user-defined Azure tags is available only in {product-title} 4.13 as a Technology Preview (TP) and for the resources created on Azure Public Cloud. User-defined Azure tags are not supported for {product-title} clusters upgraded to {product-title} 4.13.
+
+You can apply user-defined and {product-title} specific tags only to the resources created by the {product-title} installer and its core operators such as Machine api provider azure Operator, Cluster Ingress Operator, Cluster Image Registry Operator.
```

Comment:
Could we rephrase this, to mean something like tags will be applied to only those create by...?

Reply:
We can not use "will be" as per the docs hence changed a wording little bit

Reply:
Addressed